packages/framework/framework/src/subscribers/types.ts

```diff
@@ -0,0 +1,16 @@
+import { Event, MedusaContainer } from "@medusajs/types"
```

Comment:
question: Do we keep types internal to the package?

Reply:
The types will be part of a completely separate work where we will identify what needs to be part of the framework and what will not. Unless obvious typings 😅 otherwise it can create some issues and breaking changes that could be too early

Reply:
same from what I am currently doing, we are moving to get them out of the way, but we might decide to maybe have a more generic way of managing resources and then provide a front facing opinionated consume way for medusa in order to split what is fully opinionated and what needs to be generic, but that will be later discussions 👍